Sources/Metal/concat.metal

```diff
@@ -9,7 +9,27 @@
 #include <metal_stdlib>
 using namespace metal;
 
-#define INPUT_TEXTURES_COUNT 10
+/********************** Concat MARCO **********************/
+
+#define INPUT_TEXTURES_COUNT (10)
```

Comment:
And this creates a dependency to the macros that go from 0 to 9 below. Maybe don't refactor it, but add a comment to the other references instead.